templates/hooks/alias/sdk_read_many_pre_set_output.go.tpl

```diff
@@ -0,0 +1,12 @@
+// Filter resulting aliases, matching only the one with the name in the spec
+matchingAliases := []*svcsdk.AliasListEntry{}
+for _, elem := range resp.Aliases {
+  if elem.AliasName == nil || r.ko.Spec.Name == nil {
+    continue
+  }
+  
+  if *elem.AliasName == *r.ko.Spec.Name {
+    matchingAliases = append(matchingAliases, elem)
+  }
+}
+resp.Aliases = matchingAliases
```

Comment:
for the template code, you may consider using TAB characters and indenting the code to the right level. I know that `go fmt` cleans all that up when we run `make build-controller` so not a huge deal.